src/jmcomic/api.py

```
@@ -26,22 +28,25 @@ def download_batch(download_api,
     if option is None:
         option = JmModuleConfig.option_class().default()
 
-    result = set()
+    result = BatchResult()
 
     def callback(*ret):
         result.add(ret)
 
+    def _safe_download(aid):
+        """batch 内部的单任务包装：确保异常被收集而非静默丢失"""
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)